Users

mbudde opened the pull request:
I'm opening this pull request so we have a place to discuss the changes.

aossowski committed:
groups

aossowski committed:
group form - new and edit

mbudde commented on static/js/viewworld/views.js:
```diff
@@ -255,6 +255,156 @@
+    },
+
+    deleteSelected: function(){
+      $.each(this.$el.find('input:checked'), function(){
```
I'd prefer if we only use the iteration functions from Underscore.js (i.e. `_.each` instead of `$.each`).

aossowski replied:
OK, no problem

mbudde commented on static/js/viewworld/app.js:
```diff
@@ -44,11 +47,31 @@
     },
 
+    groups: function() {
+      tree = new ViewWorld.Views.GroupsView({collection: new ViewWorld.Models.Groups});
```
You instantiate a new `Groups` collection three places (here, `ViewWorld.app.groups` and in the `GroupTree` model). Is there a reason you can't use the global one everywhere?

aossowski replied:
I refactored the code to use only the global one

mbudde commented on static/js/viewworld/models.js:
```diff
@@ -145,6 +145,64 @@
+
+  }),
+
+  Models.GroupTree = Backbone.Model.extend({
```
I think this should extend `Collection` instead and use `this.model` instead of instantiating it's own model.

aossowski committed:
two and three column layouts

aossowski committed:
users

mbudde committed:
Change JSON for a single group

The returned data only need to be wrapped in a dict when the data
otherwise would have an array as the toplevel object.

See http://haacked.com/archive/2009/06/25/json-hijacking.aspx

mbudde committed:
Convert GroupTree model to a collection and join with Groups collection

mbudde committed:
Style fixes
